src/pip/_internal/wheel.py

```diff
@@ -818,9 +818,11 @@ def _build_one_inside_env(self, req, output_dir, python_tag=None):
                 builder = self._build_one_pep517
             else:
                 builder = self._build_one_legacy
-            if builder(req, temp_dir.path, python_tag=python_tag):
+
+            built_wheel = builder(req, temp_dir.path, python_tag=python_tag)
```

Comment:
I would choose a variable that helps the reader know whether it's a filename or path.